src/redash_stmo/integrations/iodide/bundle/IodideButton.jsx

```diff
@@ -101,23 +101,29 @@ export default function init(ngModule) {
         this.origRender();
 
         const buttonContainerId = 'explore-in-iodide-container';
+        const queryID = window.location.href.match(
```

Comment:
could do something like `const params = new URLSearchParams(window.location.search);params.get('queryId');...` unless the /queries/ path is somehow important

Reply:
Good catch! This is catching the query ID, which isn't a query parameter but rather part of the path. That being said, I can probably match against `window.location.path` instead of `window.location.href`.